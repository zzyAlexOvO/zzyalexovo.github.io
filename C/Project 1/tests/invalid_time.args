Wirragulla Cardiff "10:59:00"