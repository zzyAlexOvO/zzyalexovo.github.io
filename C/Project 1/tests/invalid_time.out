The next train to Cardiff from Wirragulla departs at 11:01:23
